package/yast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Dec 11 15:56:52 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- Use Auto Wrapping of long lines for Yast2::Popup (bsc#1179893)
```

Comment:
As commented on IRC, this PR does not fix bsc#1179893. Maybe we need to use another bug.

Reply:
As Josef commented on IRC, now bsc#1179893 should be fixed because *Report* was adapted.